c9-calculator.rb

```diff
@@ -0,0 +1,4 @@
+# calculator program
+# beep boop computers
+# 1 + 1 is 2
+# Making changes
```

Comment:
You could clean up this big `if`/`elsif`/`else` statement using `case`. That might look something like this:

```ruby
case operation
when '+', 'add'
  result = add(num1, num2)
when '-', 'sub'
  result = subtract(num1, num2)
# ... multiply and divide
end
```